pkg/services/live/live.go

```diff
@@ -110,9 +111,11 @@ func ProvideService(plugCtxProvider *plugincontext.Provider, cfg *setting.Cfg, r
 	// things. For example Node allows to publish messages to channels from server
 	// side with its Publish method.
 	node, err := centrifuge.New(centrifuge.Config{
-		LogHandler:         handleLog,
-		LogLevel:           centrifuge.LogLevelError,
-		MetricsNamespace:   "grafana_live",
+		LogHandler: handleLog,
+		LogLevel:   centrifuge.LogLevelError,
+		Metrics: centrifuge.MetricsConfig{
+			MetricsNamespace: "grafana_live",
```

Comment:
in 5x, there are many more metrics options... but this keeps things similar to what we had